app/scripts/contentscript.js

```diff
@@ -276,3 +370,19 @@ function redirectToPhishingWarning(data = {}) {
   const querystring = new URLSearchParams({ hostname, href, newIssueUrl });
   window.location.href = `${baseUrl}#${querystring}`;
 }
+
+const start = () => {
+  if (
+    window.location.origin === phishingPageUrl.origin &&
+    window.location.pathname === phishingPageUrl.pathname
+  ) {
+    setupPhishingStream();
+  } else if (shouldInjectProvider()) {
+    if (!isManifestV3()) {
```

Comment:
Quick note for when you rebase -- `isManifestV3` has become a boolean, so you no longer need the `()`